### PR TITLE
Move mxe prebuilt binaries to 97c0fbfd

### PR DESCRIPTION
**Pull request by salvadorcunat**
Modify prebuilt mxe binaries for smtk2ssrf build to match binaries used for subsurface build.  

Signed-off-by: Salvador Cuñat <salvador.cunat@gmail.com>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [X] Build system change
- [ ] Documentation change
- [ ] Language translation

### Related issues:

The binaries are currently untested as windows build is failing, but seem unlikely to cause issues.

### Mentions:

@dirkhh

### CODE REVIEW

File: core/qt-ble.cpp [hunk 1]
```diff
@@ -366,13 +366,15 @@ dc_status_t BLEObject::setupHwTerminalIo(QList<QLowEnergyCharacteristic> allC)
...
 
+#if !defined(Q_OS_WIN)
 // Bluez is broken, and doesn't have a sane way to query
 // whether to use a random address or not. So we have to
 // fake it.
 static int use_random_address(dc_user_device_t *user_device)
 {
 	return IS_SHEARWATER(user_device) || IS_GARMIN(user_device);
 }
+#endif
```
Comment (neolit123) [inline core/qt-ble.cpp]: hi @salvadorcunat 
i think this PR picked some extra commits.
could you please double check?

Comment (salvadorcunat) [reply core/qt-ble.cpp]: hi @neolit123.
This is not for master, but for fixTravisWindows branch, where @dirkhh is testing new mxe binaries.
It's the first time I send anything to a branch other than master, I think I've made something wrong while building the PR. Does this have any sense?
When I saw the PR last night against the fixTravisWindows branch, it was only my commit without Dirk's.

Comment (salvadorcunat) [reply core/qt-ble.cpp]: Hi @neolit123 .
I've followed your git workflow and it seems to have worked.  Please take a look, I think the PR is correct now.
Best regards.

**Comment by neolit123** [comment]
@salvadorcunat 
thanks the change looks good.

i'm going to merge this even if our CI is currently failing. seems for other reasons.

**Comment by dirkhh** [comment]
Just got logged in from my mom's house :-)
Not sure I would have merged this right now, but it's ok, no harm done...

**Pull request merged**